test/inductor/test_cpu_select_algorithm.py

```diff
@@ -235,6 +239,16 @@ def __init__(self, bias, epilogue, other):
             def forward(self, x):
                 return self.epilogue(self.linear(x))
 
+        # TODO: debug utils, safe to remove in Oct 2024
+        log.warning(
+            f"DEBUG: torch.backends.mkl.is_available() is {torch.backends.mkl.is_available()}, "  # noqa: G004
+            f"torch.ops.mkldnn._is_mkldnn_fp16_supported() is {torch.ops.mkldnn._is_mkldnn_fp16_supported()}, "
+            f"torch.ops.mkldnn._is_mkldnn_bf16_supported() is {torch.ops.mkldnn._is_mkldnn_bf16_supported()}, "
+            f"inductor_config.freezing is {inductor_config.freezing}, "
+            f"mkldnn._is_mkldnn_acl_supported() is {torch.ops.mkldnn._is_mkldnn_acl_supported()}, "
+            f"torch._C.has_mkl is {torch._C.has_mkl}."
+        )
```

Comment:
Since it is only used for debugging fbcode thing, can we wrap it with `is_fbcode()` check?